src/features/landing/components/ForWhoSection/styles.ts

```
@@ -1,5 +1,34 @@
+  width,
+  height,
+  [media.md]: { width: '0.45rem' },
+});
+
+
+const arrowCommon = {
+  position: 'absolute',
+  marginTop: '4.2rem',
+  [media.sm]: { marginTop: '4.4rem' },
+  [media.md]: { marginTop: '4.9rem' },
+};
```

ESLint

[error] Expected arrowCommon to have a type annotation.

(@typescript-eslint/typedef)